From 4116b687e6c43725a2c3ceda053dcf4c0421e49e Mon Sep 17 00:00:00 2001
From: Robin Gong <yibin.gong@nxp.com>
Date: Thu, 11 Oct 2018 20:15:12 +0800
Subject: [PATCH 4808/5242] MLK-19870-2: mfd: bd71840: add bd71840 pmic

commit  bf1929b787c60ec46eed4036ca9eee67036f03b0 from
https://source.codeaurora.org/external/imx/linux-imx.git

Add bd71840 driver name to sync the product name on board.

Signed-off-by: Robin Gong <yibin.gong@nxp.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 drivers/mfd/bd71837.c |    2 ++
 1 file changed, 2 insertions(+)

diff --git a/drivers/mfd/bd71837.c b/drivers/mfd/bd71837.c
index 1f24d07..f21ec4a 100644
--- a/drivers/mfd/bd71837.c
+++ b/drivers/mfd/bd71837.c
@@ -136,6 +136,7 @@ static bool is_volatile_reg(struct device *dev, unsigned int reg)
 #ifdef CONFIG_OF
 static struct of_device_id bd71837_of_match[] = {
 	{ .compatible = "rohm,bd71837", .data = (void *)0},
+	{ .compatible = "rohm,bd71840", .data = (void *)0},
 	{ },
 };
 MODULE_DEVICE_TABLE(of, bd71837_of_match);
@@ -275,6 +276,7 @@ static int bd71837_i2c_remove(struct i2c_client *i2c)
 
 static const struct i2c_device_id bd71837_i2c_id[] = {
 	{ "bd71837", 0 },
+	{ "bd71840", 0 },
 	{ }
 };
 MODULE_DEVICE_TABLE(i2c, bd71837_i2c_id);
-- 
1.7.9.5

